pkg/customers/approval_ofac.go

```diff
@@ -104,6 +104,8 @@ func (s *OFACSearcher) storeCustomerOFACSearch(cust *client.Customer, requestID
 }
 
 func AddOFACRoutes(logger log.Logger, r *mux.Router, repo CustomerRepository, ofac *OFACSearcher) {
+	logger = logger.WithKeyValue("package", "ofac")
```

Comment:
I'm not sure about `"package":"ofac"` here. "ofac" was used as a key before and I don't know how to log it now. But using it here as package maybe misleading...

any ideas @adamdecaf @vxio ?

Reply:
What about something like `customers-ofac`? Granted it's not actually called OFAC anymore... 🤷‍♂️

Reply:
I'd just remove this.

Reply:
If I know how we are going to use it later (how filtering is going to work)... :)

Reply:
You mean how we're going to search logs, set up alerting, etc?

I imagine queries will just be something like `{package="packageName"} |= "some-search-params" ...`. Is this to what you refer? ;)

Reply:
I think that we should extract it into ofac package... :) But it's out of this PR. For now changing to {"package": "customers"} and closing this issue :D

Reply:
Oh I gotcha. I was thinking that as well. This feels out of place.

I'd agree with making that change for now (the one in your comment) so we can get this merged. The longer this PR lives the harder it'll be to merge :(

Reply:
Yeap. I mean if we filter by "customers" then "customers-ofac" may not be included and so on.

I also have one more **BIG** PR that it ready and is waiting for this one... and ...rebase 😭

Reply:
😭 indeed